types/africastalking/index.d.ts

```diff
@@ -38,29 +39,74 @@ interface SMS {
         phoneNumber: string;
         checkoutToken: string;
     }) => Promise<{
-        "description": "Success" | "Failed";
-        "token": string;
+        description: "Success" | "Failed";
+        token: string;
     }>;
 }
 
 interface TOKEN {
     generateAuthToken: () => Promise<{
-        "description": "Success" | "Failed";
-        "token": string;
+        description: "Success" | "Failed";
+        token: string;
     }>;
     createCheckoutToken: (phoneNumber: string) => Promise<{
         description: "Success" | "Failed";
         token: string;
     }>;
 }
 
+interface CallEntry {
+    phoneNumber: string;
+    status: "Queued" | "InvalidPhoneNumber" | "DestinationNotSupported" | "InsufficientCredit";
+    sessionId: string;
+}
+
+interface NumQueuedCallsEntry {
+    phoneNumber: string;
+    queueName: string;
+    numCalls: number;
+}
+interface VOICE {
+    /**
+     * This is used to make outbound calls
+     *
+     * @param callFrom Your Africa’s Talking phone number (in international format i.e. +XXXYYYYYY)
+     * @param callTo An array of entries each corresponding to an individual phone number and their status.
+     * @param clientRequestId Variable sent to your Events Callback URL that can be used to tag the call
+     */
+    call: (options: {
+        callFrom: string;
+        callTo: string[];
+        clientRequestId?: string;
+    }) => Promise<{
+        entries: CallEntry[];
+        errorMessage?: string;
+    }>;
+
+    getNumQueuedCalls: (options: {
+        phoneNumbers: string;
+    }) => Promise<{
+        status: string;
+        entries: NumQueuedCallsEntry[];
+        errorMessage: string;
+    }>;
+
+    uploadMediaFile: (options: {
+        phoneNumbers: string;
+        url: string;
+    }) => Promise<any>;
+}
+
 interface AfricasTalking {
     SMS: SMS;
     TOKEN: TOKEN;
+    VOICE: VOICE;
 }
 
-export = africastalking;
+declare function africastalking(options: {
+    username: string;
+    apiKey: string;
+}): AfricasTalking;
 
-declare function africastalking(
-    options: { username: string; apiKey: string },
-): AfricasTalking;
+declare namespace africastalking {}
```

Comment:
Why is this here? It's empty and couldn't do anything.

Reply:
In the previous method I found that I could get the types to work in the example file but this one does

Reply:
Did you mean to declare the interfaces inside the "africastalking" namespace, so that they are also exported and can be imported and used elsewhere?

Reply:
@gabritto  Its so that I can be able to get the types in the africastalking tests file..for some reason the earlier method didnt allow me to access types in the test folder but with that extra change they were now visible.

Reply:
Ok, I still think the interfaces declared here should be *inside* the namespace. Can you make that change?